cranelift/codegen/src/isa/x64/inst/emit.rs

```diff
@@ -3968,12 +3850,7 @@ pub(crate) fn emit(
             if *is_saturating {
                 // The input was "small" (< 2**(width -1)), so the only way to get an integer
                 // overflow is because the input was too small: saturate to the min value, i.e. 0.
-                let inst = Inst::alu_rmi_r(
-                    *dst_size,
-                    AluRmiROpcode::Xor,
-                    RegMemImm::reg(dst.to_reg()),
-                    dst,
-                );
+                let inst = Inst::xor(*dst_size, RegMemImm::reg(dst.to_reg()), dst); // TODO: uninit for regalloc?
```

Comment:
This "TODO" should be ok because this is all post-regalloc so no need to worry about uninit/etc